cmd/containerd-shim-runhcs-v1/service.go

```diff
@@ -205,9 +203,7 @@ func (s *service) Kill(ctx context.Context, req *task.KillRequest) (_ *google_pr
 	ctx, span := trace.StartSpan(ctx, "Kill")
 	defer span.End()
 	defer func() {
-		if !errdefs.IsNotFound(err) {
-			oc.SetSpanStatus(span, err)
-		}
+		oc.SetSpanStatus(span, err)
```

Comment:
nit: make this one line again

Reply:
I saw that too. All the others are broken out. Want me to update all of them?

Reply:
This is the only one I saw that only calls `SetSpanStatus` that also is multi-line. Are there more?